model-optimizer/install_prerequisites/install_prerequisites.sh

```diff
@@ -181,21 +166,20 @@ find_ie_bindings() {
 }
 
 if [[ $V_ENV -eq 1 ]]; then
-    "$python_binary" -m venv "$SCRIPTDIR/../venv${postfix}"
-    source "$SCRIPTDIR/../venv${postfix}/bin/activate"
-    venv_python_binary="$SCRIPTDIR/../venv${postfix}/bin/$python_binary"
+    "$python_binary" -m venv "$HOME/venv_mo${postfix}"
+    source "$HOME/venv_mo${postfix}/bin/activate"
```

Comment:
Let's introduce additional variable to store `$HOME/venv_mo`. This sub-path is used in 3 places in the code